lib/folio_client.rb

```diff
@@ -74,4 +74,11 @@ def fetch_hrid(...)
       inventory.fetch_hrid(...)
     end
   end
+
+  def instance_status?(...)
+    TokenWrapper.refresh(config, connection) do
+      inventory = Inventory.new(self)
+      inventory.instance_status?(...)
+    end
+  end
```

Comment:
if we're passing in the status uuid we care about, should this be `instance_status=?`